libs/langchain/langchain/tools/github/prompt.py

```diff
@@ -38,12 +36,10 @@
 
 test/test.txt
 
-test contents
-"""
+test contents"""
 
 READ_FILE_PROMPT = """
-This tool is a wrapper for the GitHub API, useful when you need to read the contents of a file in a GitHub repository. Simply pass in the full file path of the file you would like to read. **IMPORTANT**: the path must not start with a slash
-"""
+This tool is a wrapper for the GitHub API, useful when you need to read the contents of a file. Simply pass in the full file path of the file you would like to read. **IMPORTANT**: the path must not start with a slash"""
```

Comment:
Unless there's a strong reason to remove the newlines at the ends of these prompt strings, could we leave them as-is? Makes it easier to isolate issues if people see regressions in their performance if we implement the minimal change in this PR

(applies to all the prompt string changes above)